tests/cli/__init__.py

```diff
@@ -1,2 +0,0 @@
-# Copyright 2022 MosaicML Composer authors
```

Comment:
Unintentional?

Reply:
moved cli test to top level since folder has one file to be consistent